internal/code/code.go

```diff
@@ -21,7 +21,7 @@ type Result struct {
 }
 
 // ?: means non-capture group
-var re = regexp.MustCompile("(?s)(?:```|~~~)(\\w+)\n(.*)\n(?:```|~~~)\n")
+var re = regexp.MustCompile("(?s)(?:```|~~~)(\\w+)\n(.*)\n(?:```|~~~)\\s?")
```

Comment:
This means that people can have trailing spaces or tabs after the three backticks? If it's optional anyway we can just remove the `\s?` and I believe everything should still work

Reply:
I'm not sure, if we leave out the `\s`, would it match things like `~~~~~`?

Reply:
Oh yea you're totally right! My bad